### PR TITLE
[Fix] Resolve Protocol.UndefinedError by using fallback

### PR DESCRIPTION
**Pull request by OldhamMade**
Fixes an issue where `:requests` is `nil` but the following lines expect an empty list.

This was a quick fix for my own project, so I haven't put together any tests yet. However, all current tests pass locally.

### CODE REVIEW

File: lib/crawly/worker.ex [hunk 1]
```diff
@@ -158,8 +158,8 @@ defmodule Crawly.Worker do
              parsed_item: Crawly.ParsedItem.t(),
              result: {:ok, :done}
   defp process_parsed_item({parsed_item, response, spider_name}) do
-    requests = Map.get(parsed_item, :requests, [])
-    items = Map.get(parsed_item, :items, [])
+    requests = Map.get(parsed_item, :requests) || []
```
Comment (oltarasenko) [inline lib/crawly/worker.ex:161]: Sorry, I don't quite understand the difference between the:
`requests = Map.get(parsed_item, :requests) || []` and 
`requests = Map.get(parsed_item, :requests, [])`

Comment (OldhamMade) [reply lib/crawly/worker.ex:161]:
```elixir
iex(1)> Map.get(%{requests: nil}, :requests, [])
nil
iex(2)> Map.get(%{requests: nil}, :requests) || []
[]
```

Comment (OldhamMade) [reply lib/crawly/worker.ex:161]: Line 164 expects a list and fails on `nil`:

```elixir
iex(1)> Enum.each(nil, &(&1))
** (Protocol.UndefinedError) protocol Enumerable not implemented for nil of type Atom
...
iex(2)> Enum.each([], &(&1))
:ok
```

Comment (oltarasenko) [reply lib/crawly/worker.ex:161]: Ah, ok. Clear now

File: mix.exs [hunk 2]
```diff
@@ -2,7 +2,7 @@ defmodule Crawly.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/oltarasenko/crawly"
-  @version "0.16.0"
+  @version "0.16.1"
```
Comment (oltarasenko) [inline mix.exs:5]: Please remove this change

**Review by oltarasenko — CHANGES_REQUESTED** [review]
Please rebase your commit and remove changes to mix.exs, so I can merge the code

**Comment by OldhamMade** [comment]
Done!

**Review by oltarasenko — APPROVED** [review]

**Pull request merged**